.github/workflows/arm64-build.yml

```diff
@@ -0,0 +1,32 @@
+name: ARM64 build
+
+on:
+  # run every day of the week at 06:00
+  schedule:
+    - cron: 0 6 * * *
+
+jobs:
+  build-arm64-docker:
+    name: Build for ARM64 on Docker
```

Comment:
ARM build takes 4h, so separated to a standalone file and runs on schedure (nightly at 6am)